### PR TITLE
Update to the latest previous versions

### PR DESCRIPTION
**Pull request by abdonrd**

### CODE REVIEW

**Comment by robdodson** [comment]
just asking @FredKSchott to take a quick look since he's more familiar with the latest dependencies

File: bower.json [hunk 1]
```diff
@@ -15,14 +15,14 @@
...
+    "polymer": "Polymer/polymer#^2.0.0-rc.3",
+    "webcomponentsjs": "webcomponents/webcomponentsjs#^1.0.0-rc.7"
   },
   "resolutions": {
-    "polymer": "^2.0.0-rc.1"
+    "polymer": "^2.0.0-rc.3"
   },
   "devDependencies": {
-    "web-component-tester": "6.0.0-prerelease.5"
+    "web-component-tester": "6.0.0-prerelease.7"
```
Comment (stramel) [inline bower.json]: I feel like this should be `"web-component-tester": "Polymer/web-component-tester#^6.0.0-prerelease.7"`

Comment (abdonrd) [inline bower.json]: Makes sense! Updated!

**Comment by FredKSchott** [comment]
LGTM! Lets merge now before things change again :)

**Pull request merged**